jwst/jump/tests/test_jump_step.py

```diff
@@ -27,6 +27,7 @@ def generate_miri_reffiles(tmpdir_factory):
     gain_model.meta.subarray.xsize = xsize
     gain_model.meta.subarray.ysize = ysize
     gain_model.save(gainfile)
+    gain_model.close()
```

Comment:
Hadn't noticed until now that these tests are generating files saved to disk. Are those being deleted later or are they just left sitting around in the test directories?

Reply:
they are written out to one of the pytest directories. i think you can override a reference file with a datamodel instead of a file path right, so these wouldn't have to be written out? i can open another pr to change this if thats the case

Reply:
Without the close all the tests were failing with an error about files not being closed. This must have been due to some other change since I haven't touched the JWST unit tests for Jump.